lib/modules/datasource/metadata-manual.ts

```diff
@@ -46,14 +46,27 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
     wagtail: 'https://github.com/wagtail/wagtail/tree/master/docs/releases',
   },
   docker: {
+    'confluentinc/ksqldb-server': 'https://github.com/confluentinc/ksql',
+    'confluentinc/ksqldb-cli': 'https://github.com/confluentinc/ksql',
+    'crossplanecontrib/provider-helm': 'https://github.com/crossplane-contrib/provider-helm',
+    'crossplanecontrib/provider-kubernetes': 'https://github.com/crossplane-contrib/provider-kubernetes',
+    'prom/blackbox-exporter': 'https://github.com/prometheus/blackbox_exporter',
```

Comment:
Can this be reordered alphabetically? :)

Reply:
done with force push, sorry I had missed this one.